src/js/components/stock-movement-wizard/CreateStockMovement.jsx

```diff
@@ -172,6 +180,38 @@ class CreateStockMovement extends Component {
     return (checkOrigin || checkDest || checkStockList);
   }
 
+  fetchInitialValues() {
```

Comment:
I think this could be named setInitialLocations or something similar, because this is not fetching anything